bin/generate-bundle.js

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env node
+const glob = require('glob');
+const path = require('path');
+const less = require('less');
+const fs = require('fs');
+const pkg = require('../package.json');
+const LessPluginCleanCSS = require('less-plugin-clean-css');
+const cleanCSSPlugin = new LessPluginCleanCSS({ advanced: true });
+const LessPluginAutoPrefix = require('less-plugin-autoprefix');
+const autoprefixPlugin = new LessPluginAutoPrefix();
+const currentDir = path.dirname(__dirname);
+const { exec } = require('child_process');
+
+// The list of directories in the dist to pull
+const dsList = ['ds4', 'ds6'];
+
+/**
+ * Main Processing class. Holds info about args passed and ds version
+ */
+class CssProcesser {
+  constructor(dsV, args) {
+    this.dsV = dsV;
+    this.args = args;
+    this.processed = [];
+    this.skipped = [];
+
+    let classDef = '';
+    if (args.scopeClass) {
+      for (let i = 0; i < args.scopeSpecificity; i++) {
+        classDef += `.${args.scopeClass}`;
```

Comment:
Set scopeClass to a variable so it doesn't have to be accessed in the loop.  (minor)